core/iwasm/aot/aot_runtime.h

```diff
@@ -225,7 +238,7 @@ typedef struct AOTModule {
 
     /* function info */
     uint32 func_count;
-    /* func pointers of AOTed (un-imported) functions */
+    /* func pointers of AOTed (imported + local) functions */
```

Comment:
how about also changing comment of L243?

Reply:
`func_type_indexes` still only stores for local(un-imported) functions.